### DISCUSSION TITLE
where are screenshots saved?

### DISCUSSION THREAD

**Discussion by bok4600** [vzhd1701/gridplayer#92] [Q&A]
i'm using the windows portable version, where are the screenshots of the video I take are stored?

**Comment by vzhd1701** [comment]
GridPlayer can't save screenshots, what are you talking about?

**Comment by bok4600** [comment]
![Screenshot 2022-09-02 100801](https://user-images.githubusercontent.com/112693587/188179416-4331ec05-682b-4eea-8fda-06fe6a77e738.png)

this

**Reply by vzhd1701** [reply]
Those are slots for saving playlist state, not screenshots.

**Reply by bok4600** [reply]
oops my bad sorry
